llvm/lib/Target/RISCV/RISCVPfmCounters.td

```diff
@@ -0,0 +1,18 @@
+//===---- RISCVPfmCounters.td - RISCV Hardware Counters ----*- tablegen -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// This describes the available hardware counters for RISCV.
```

Comment:
RISCV -> RISC-V

We should adhere as much as possible to the branding guidelines https://riscv.org/about/risc-v-branding-guidelines/